Packs/ServiceNow/ReleaseNotes/2_5_58.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### ServiceNow v2
+
+- Added the *add_as_entry* argument to the *servicenow-get-ticket-notes* command to optionally add the notes and work_notes as an entry to the War Room.
```

Comment:
Recommend:
Added the *add_as_entry* argument to the ***servicenow-get-ticket-notes*** command to optionally add ticket notes and work notes as notes in the War Room.